### DISCUSSION TITLE
Prevent layout shift when toggling scrollbar on Window | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#73] [Announcements]
# Prevent layout shift when toggling scrollbar on Window | Leo's dev blog – stories, insights, and ideas

When toggling the scrollbar on the Window, the layout shifts. This is because the scrollbar is added to the Window. How to prevent this with CSS?

https://www.leohuynh.dev/blog/prevent-layout-shift-while-show-hide-scrollbar-on-window

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
